simtools/utils/general.py

```diff
@@ -3,6 +3,7 @@
 """
 
 import copy
+import json
```

Comment:
I think this is something we need to decide on. Should we agree to always use the astropy json package instead? There are some issues regarding astropy units if one reads a json file with the normal json package (though one can test the impact more closely).

Reply:
Note entirely sure - we use the astropy yaml package because of some extensions (which is actually not ideal - as astropy is using pyaml which does not allows follows the yaml standard), but we do not use any of those extensions for json. I would advocate the simpler solution (using the base package), unless anything else is required.

Reply:
ok! I think I mixed up, since astropy has a yaml package but doesn't seem to have a json similar